Documentation/architecture.md

```diff
@@ -58,7 +58,7 @@ You should avoid public access to etcd and instead run fleet [from your local la
 
 ## Securing fleetd
 
-It is also recommended to run fleetd under separate `fleet` user and group, and set the permissions of the fleetd API's listening Unix socket to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. Since the fleet daemon uses [D-Bus][d-bus] to communicate with systemd it is necessary to create a [`polkit(8)`][polkit] rule to allow fleetd to communicate with systemd:
+Starting from v216 systemd supports [`polkit(8)`][polkit] rules to verify access for unprivileged users. Thus it is recommended to run fleetd under separate `fleet` user and group, and set the permissions of the fleetd API's listening Unix socket to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. The polkit rule below grants `fleetd` daemon ran by unprivileged `fleet` user to perform actions with systemd through [D-Bus][d-bus]:
```

Comment:
Do they need to be the fleet user, or just in the fleet group? It's a little ambiguous from the current wording.